src/components/layout/footer.js

```diff
@@ -10,7 +10,7 @@ import BottomSocialSection from './footer/bottom-social-wrapper'
 import { DerivStore } from 'store'
 // TODO: (discussion) make footer pure component, and move usage of footer to custom
 
-const Footer = ({ type, is_ppc, is_ppc_redirect, academy }) => {
+const Footer = ({ type, is_ppc, is_ppc_redirect, academy, no_footerlinks }) => {
```

Comment:
base on our code styles this should be like this => no_footer_links

Reply:
done